bson/src/main/org/bson/BsonObjectId.java

```diff
@@ -28,6 +28,13 @@
     private final ObjectId value;
 
     /**
+    * Construct a new instance with a new {@code ObjectId}.
```

Comment:
Is this really warranted?  It's a minor convenience only.  If we do end up keeping it, needs an @since annotation